controllers/submariner/route_agent_resources.go

```diff
@@ -110,7 +110,6 @@ func newRouteAgentDaemonSet(cr *v1alpha1.Submariner, name string) *appsv1.Daemon
 							Name:            name + "-init",
 							Image:           getImagePath(cr, opnames.RouteAgentImage, names.RouteAgentComponent),
 							ImagePullPolicy: images.GetPullPolicy(cr.Spec.Version, cr.Spec.ImageOverrides[names.RouteAgentComponent]),
-							Command:         []string{"submariner-route-agent.sh"},
```

Comment:
This removal is harmless, but was it intended as part of this PR?

Reply:
It's cleanup that was missed before but technically should be in another PR.  I'll remove it.